src/abinitio.py

```diff
@@ -1104,3 +1104,21 @@ def compute_objective_part(SPX,QQ0,Q0,Z,a,n,energy=False,subtract_mean=False,div
         raise RuntimeError('Please pass either 0, 1, 2 to divide')
     return X2
 
+def plot_mm_vs_qm(M, Q, title=''):
+    import matplotlib.pyplot as plt
```

Comment:
I think python use a cache for imported modules, so multiple calls of this function will not be affected. https://docs.python.org/2/reference/simple_stmts.html#the-import-statement